### ISSUE TITLE
Add support for RaZberry

### ISSUE THREAD

**Issue by Isaksson** [OpenELEC/OpenELEC.tv#2179]
The RaZberry turns every Raspberry PI into a Z-Wave Home Automation Gateway.
Here is a link to the RaZberry homepage http://razberry.z-wave.me/

I would like to be able to install the software on openelec os.
Here is a link to the installation script  http://razberry.z-wave.me/install

I have also started a thread in there forum at http://en.z-wave.me/forums/razberry

Answer from RaZberry:

With Openelec it will work too, but you need to add Z-Way/RaZberry to startup script manually. Also the configuration f ttyAMA0 is not needed on OpenElec, so just skip the error messages.

To summarize, the software can be installed in /opt and will work, but init scripts will not - do them yourself

It would be great to wrap our software in an Openelec plugin.
